src/Nav/Nav.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import "./Nav.css";
+
+const Nav = () => {
+    return (
+        <nav className="main">
+            <div className="content-wrapper">
```

Comment:
mate I think the className `content-wrapper` is already being used on Pokemon.js, so this will make your styles topsy turvy mate! Might be a good idea to prefix all Nav class-names with `nav-` like `nav-content-wrapper`